Radzen.Blazor/RadzenDialog.razor.cs

```diff
@@ -0,0 +1,256 @@
+using Microsoft.AspNetCore.Components;

```

Comment:
Please leave this code in RadzenDialog.razor for now as it is easier to track the changes this way. We may separate them later.